styles/rule.css

```diff
@@ -103,3 +103,19 @@ a {
   border-radius: 2px;
   background-color: var(--vscode-textBlockQuote-background);
 }
+
+.other-context-list {
+  list-style: none;
+}
+
+.other-context-list .do::before {
+  content: "\2713";
+  color:green;
+  padding-right:5px;
+}
+
+.other-context-list .dont::before {
+  content: "\2717";
+  color:red;
+  padding-right:5px;
+}
```

Comment:
Could we try with CSS variables for the colors, to make them theme-aware?

E.g I noticed that we should be able to use these in the webview:

```css
    --vscode-testing-iconFailed: #f14c4c;
    --vscode-testing-iconErrored: #f14c4c;
    --vscode-testing-iconPassed: #73c991;
```

Reply:
Great idea, tested and it works.